ansible/README.md

```diff
@@ -73,20 +73,32 @@ In all instructions, replace `<openwhisk_home>` with the base directory of your
 
 #### Setup
 
-This step needs to be done only once per development environment. It will generate configuration files based on your local settings. Notice that for the following playbook you don't need to specify a target environment as it will run only local actions.
-After the playbook is done you should see a file called `db_local.ini` in your `ansible` directory. It will by default contain settings for a local ephemeral CouchDB setup. Afterwards, you can change the values directly in `db_local.ini`.
-
-#####  Ephemeral CouchDB
-
-If you want to use the ephemeral CouchDB, run this command
+The following step must be executed once per development environment.
+It will generate the `hosts` configuration file based on your environment settings.
 
 ```
 ansible-playbook -i environments/<environment> setup.yml
 ```
 
-#####  Persistent CouchDB
+The default configuration does not replicate core components.
```

Comment:
use something else than `replicate`, 
The default configuration does not run multiple instances of core components (i.e. controllers, invokers, kafka, etc..)
Or something similar to that extend `multiple instances`